lib/wickedbase.pm

```diff
@@ -235,7 +235,8 @@ sub ping_with_timeout {
             if ($args{count_success} > 1) {
                 my $cnt = $args{count_success} - 1;
                 $ping_command =~ s/\s-c\s1\s+/ -c $cnt /;
-                validate_script_output($ping_command, sub { /\s+0% packet loss/ });
+                my $ping_out = script_output($ping_command, proceed_on_failure => $args{proceed_on_failure});
+                record_info('WARNING PING FAILED', $ping_out) unless ($ping_out =~ /\s+0% packet loss/ )
```

Comment:
Should it be `record_softfailure` ?
I actually would vote for specifying a threshold in `%` of failed packages.

Reply:
1. there should be no softfailures without bug ID. 
2.  theshold added

Reply:
ok